### ISSUE TITLE
Compiler Warnings in build.properties cause IAE in OpenHotspotInEditor Action

### ISSUE THREAD

**Issue by GoogleCodeExporter** [hansriezebos/projectusus#20]
```
please see attached screenshot for details. I think this can be fixed quickly.
```

Original issue reported on code.google.com by `stg.info...@gmail.com` on 12 Dec 2009 at 1:57

Attachments:
- [build.properties_causes_exception.png](https://storage.googleapis.com/google-code-attachments/projectusus/issue-20/comment-0/build.properties_causes_exception.png)

**Comment by GoogleCodeExporter** [comment]
Original comment by `leiffren...@googlemail.com` on 15 Dec 2009 at 5:29
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.4. Thanks :-)
```

Original comment by `leiffren...@googlemail.com` on 15 Dec 2009 at 7:04
- Changed state: **Fixed**
